silkworm/silkrpc/core/storage_walker.cpp

```diff
@@ -89,7 +89,7 @@ Task<ethdb::SplittedKeyValue> next(ethdb::SplitCursor& cursor, uint64_t number,
 }
 
 Task<void> StorageWalker::walk_of_storages(
-    uint64_t block_number,
+    BlockNum block_number,
```

Comment:
The same change should be applied also in `storage_walker.hpp`.

Reply:
ok